### PR TITLE
molecular/ardock component scratch

### PR DESCRIPTION
**Pull request by glaunay**
Providing access to our app which is a webserver.
Propose to define a custom component to embed a iframe to our webervice.

### CODE REVIEW

**Comment by glaunay** [comment]
Needs reviewing and fixing !
Open to discussion

File: src/assets/config_files/usecases.json [hunk 1]
```diff
@@ -47,6 +47,26 @@
...
+        "maturity": ["beta"],
+        "disabled": false,
+        "picture": {
+          "src": "https://raw.githubusercontent.com/antonelepfl/usecases/dev/src/assets/images/ardock.png"
+        },
+        "next": "ta_form",
+        "dataprotected": true,
+        "files": [
+          {
+            "entryname": "ArDock",
```
Comment (antonelepfl) [inline src/assets/config_files/usecases.json]: This is going to be the 'nav item' that the user will get into the Collab. Do you think that name is representative enough? Could it be Detection Interaction? as you prefer.

File: src/assets/config_files/usecases.json [hunk 2]
```diff
@@ -47,6 +47,26 @@
           }
         ]
       },
+      {
+        "title": "Predict protein interaction surface with the ARDOCK server",
+        "description": "In silico analysis of your protein surface using molecular probes to reveal potential interaction sites",
+        "experience": ["all"],
+        "maturity": ["beta"],
+        "disabled": false,
```
Comment (antonelepfl) [inline src/assets/config_files/usecases.json]: You can add also some Collaborators if you want (it's optional)
Like https://github.com/antonelepfl/usecases/pull/214/files#diff-792ef00d694f11cf071142152aecd528R160

**Comment by antonelepfl** [comment]
The end result after my modifications should be something like:

Screenshot 2019-05-28 at 16 07 22

As Alex mentioned the space is reduced so probably some things could be smaller to give priority to the main content.

**Comment by lbologna** [comment]
Hello,
@glaunay is there any news on this ticket? May we help in some way to finalize it or if it has to be finalized at a later time, close the ticket for the moment?

**Comment by alex4200** [comment]
@antonelepfl Can you have a look at this PR please? Can it be merged now?

**Comment by antonelepfl** [comment]
I'll merge this but it would be nice as mentioned [above](https://github.com/antonelepfl/usecases/pull/210#issuecomment-496533058) in the comments the frame is a bit big and it reduce space of the main content. This could be an improvement for the future.

**Comment by glaunay** [comment]
Tx, we'll look into it.

**Pull request merged**